src/doom/DoomMain.java

```diff
@@ -1523,7 +924,22 @@ protected void GenerateTitle() {
             title.append(VERSION%100);
             title.append("                           ");
             break;
-
+        case freedoom1:
+            title.append("                            ");
+            title.append("FreeDoom: Phase 1          v");
+            title.append(VERSION/100);
+            title.append(".");
+            title.append(VERSION%100);
+            title.append("                           ");
+            break;
+        case freedoom2:
+            title.append("                            ");
+            title.append("FreeDoom: Phase 2          v");
+            title.append(VERSION/100);
+            title.append(".");
+            title.append(VERSION%100);
+            title.append("                           ");
+            break;
```

Comment:
The strings are in the DEH patch that's in the IWAD. We must add support for this later.